peeps/PEEP-003.md

```diff
@@ -0,0 +1,59 @@
+# PEEP-003: Specify pipenv options from configuration file
+
+This PEEP describes an addition that would allow configuring `pipenv` options via configuration file.
+
+☤
+
+The discussion in [#2778](https://github.com/pypa/pipenv/issues/2778#issuecomment-417966352) suggested that a new PEEP needed to be created to address the feature discussed in the github issue.
```

Comment:
As @jxltom mentioned, A PEEP needs to be self-contained, so in this document you need to assume there is no preexisting issue and that nobody cares about reported problems.  This is a document to describe a change you want to see in pipenv, so you need to describe the change in full, what it would do, why you are proposing it, what alternatives you've considered, and you can mention the fact there have been issues reported that led you to propose this. I have a proposal I was working on that I can upload which may come close, it might help you consolidate your ideas here.

Essentially we want as few unnecessary words as possible and we want to answer as many questions as possible ahead of time